src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,81 @@
+package duke.tasks;
+
+import duke.functions.Ui;
+
+/**
+ * Task class for tasks that users input into the Duke bot.
+ * @author lauralee
+ */
+public class Task {
+    private String name;
+    private String type;
+    private int status;
+    public static int numberTasks = 0;
```

Comment:
Maybe this constant variable should be in all uppercase with underscore to separate words? (eg. `NUMBER_TASKS`)